db/fixtures/chargeback_rates.yml

```diff
@@ -12,6 +12,7 @@
       :per_time: daily
       :per_unit: megahertz
       :measure: Hz Units
+      :type_currency: Dollars
```

Comment:
Where do we update existing chargeback_rates where type_currency was previously nil?